tests/api/event-services/work-request.test.js

```diff
@@ -29,7 +29,7 @@ describe('handleWorkRequest', () => {
       socketId: '6789',
       experimentId: 'my-experiment',
       timeout: '2001-01-01T00:00:00Z',
-      body: { name: 'GetEmbedding', config: {} },
+      body: { name: 'GetEmbedding', config: { minimumDistance: 0.3, distanceMetric: 'euclidean' } },
```

Comment:
The type parameter is missing on these requests, and it should be "tsne/umap/pca"

Reply:
Can you add it as a parameter, or is the test relying on this not being set?

Reply:
Good catch @ogibson-biomage, there is an error in the schema that makes `type` not required.